src/service/video-manager-impl.js

```diff
@@ -493,6 +538,39 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @param {boolean} isLandscape
+   * @private
+   */
+  orientationChanged_(isLandscape) {
+    // Put the video in/out of fullscreen depending on orientation
```

Comment:
interesting, it is technically correct. Can we ensure at least we don't go to fullscreen on rotation if `viewerForDoc(this.ampdoc_).isVisible()` is `false`?